docker-compose.platform.yml

```diff
@@ -33,7 +33,11 @@ services:
       - DASHPAY_CONTRACT_ID=${PLATFORM_DASHPAY_CONTRACT_ID}
       - DASHPAY_CONTRACT_BLOCK_HEIGHT=${PLATFORM_DASHPAY_CONTRACT_BLOCK_HEIGHT}
       - NODE_ENV=${ENVIRONMENT:?err}
-      - LOGGING_LEVEL=${PLATFORM_DRIVE_ABCI_LOG_LEVEL:?err}
+      - LOG_STDOUT_LEVEL=${PLATFORM_DRIVE_ABCI_LOG_STDOUT_LEVEL:?err}
+      - LOG_PRETTY_FILE_LEVEL=${PLATFORM_DRIVE_ABCI_LOG_PRETTY_LEVEL:?err}
+      - LOG_PRETTY_FILE_PATH=${PLATFORM_DRIVE_ABCI_LOG_PRETTY_FILE_PATH:?err}
```

Comment:
I guess here file paths must be static (inside the container). The paths from mn config supposed to be for the host machine, right?